api/api.go

```diff
@@ -70,18 +87,44 @@ func (w Wrapper) AuthenticateWithIRMA(ctx echo.Context) error {
 }
 
 func (w Wrapper) GetIRMAAuthenticationResult(ctx echo.Context, sessionToken string) error {
+	// current customerID
+	bearerToken := ctx.Request().Header.Get(echo.HeaderAuthorization)
+	token, err := w.Auth.ValidateJWT([]byte(bearerToken[7:]))
```

Comment:
line 92 til 102 is duplicate

Reply:
only the extraction is dupl. The middleware does an extra check that cannot be done by the IRMAAUthentication